ui/pages/configuration/filters.js

```diff
@@ -0,0 +1,44 @@
+import { NoSsr, Paper, withStyles } from "@material-ui/core";
+import MesheryFilters from "../../components/Filters";
+import { updatepagepath } from "../../lib/store";
+import { connect } from "react-redux";
+import { bindActionCreators } from 'redux'
+import Head from 'next/head';
+import { getPath } from "../../lib/path";
+
+const styles = {
+  paper: {
+    maxWidth: '90%',
+    margin: 'auto',
+    overflow: 'hidden',
+  }
+}
+
+class newFilters extends React.Component {
```

Comment:
It's good practice to start the className in Caps.

Reply:
@Nikhil-Ladha Also, there is another file called `Filter.js` but it's with F in Caps here at, https://github.com/layer5io/meshery/blob/master/ui/components/Filters.js.
I thought it would be better to rename the file created in this PR to something else, so to avoid any confusions along with the className. What's your opinion on this?

Reply:
I am talking about the className and not the file name. The className for the component is `MesheryFilters`, so it won't be an issue.

Reply:
Okay, gotcha 😶

Reply:
Ohh sorry. I think you meant something else.......I always tend to dot this 😞 .
The name that you give the file under `pages` directory will be the path that will be created in case of NextJs, so we can't rename the file to something else.